internal/host/host.go

```diff
@@ -321,6 +322,24 @@ func (m *Manager) updateInventory(ctx context.Context, cluster *common.Cluster,
 		return common.NewApiError(http.StatusNotFound, err)
 	}
 
+	if m.Config.BootstrapHostMAC != "" {
```

Comment:
This is OK and should work.  Would you rather have the logic here where it is executed approximately once a minute upon inventory update, or perhaps in setBootstrapHost() in inventory.go?

Reply:
Moving this logic to `setBootstrapHost` makes sense.

The problem with that is that there is a situation where the node that should be bootstrap might not have a master role.
Doing the bootstrap assignment here reduces the probability of that (the user might still change the node role and click install before the host sends inventory again).

I guess we should also add code that blocks setting the host role as worker in case the host has the `BootstrapHostMAC`

Reply:
I added a condition to check that the host is not already set as the bootstrap, so that we don't execute the DB calls more than once.

I initially tried it in setBootstrapHost(), but then moved it here for the reason Eran mentioned - at that point it is too late to change the Role because installation has already started.

> I guess we should also add code that blocks setting the host role as worker in case the host has the BootstrapHostMAC

I think this could be more generic: don't allow setting the role to Worker if the host is marked as the bootstrap.